hack/verify-govet.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For use with Go 1.12+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TRAVIS_GO_VERSION=${TRAVIS_GO_VERSION:-}
```

Comment:
Can we please drop down to use `go version` when `TRAVIS_GO_VERSION` is not set?

Reply:
Yeah, but then I have to parse it... =)

`go version` gives me `go version go1.12.1 darwin/amd64`, whereas `TRAVIS_GO_VERSION` just gives me `1.12` or `1.12.1` or `1.12.x`.

I was trying to avoid parsing it by making the default behavior be, if that env var is not set, just run the 1.12+ version (which is `go vet`), as most K8s devs should be on 1.12 anyways. But maybe that's a faulty assumption?

Reply:
If I need to parse it, I might as well parse it all the time. It wouldn't be that hard... I can do that.